specs/examples/README.md

```diff
@@ -0,0 +1,157 @@
+- Issue templates should be updated to include the Stackblitz urls to make it easier to report issues
+- Types of examples (just ideas / not mandatory by this spec)
+  - Specific-component usage - Detailed examples for some more detailed configuration of specific components
+  - Stress tests - Lots of components, large data sets, fast updates. Would align well with Playwright-based perf tests we have experimented with.
+  - Examples for unsupported frameworks, i.e. example React app, vue.js app, etc. leveraging the web-component APIs directly. While we don't provide first class well-integrated support, having smoke test examples and quickly stackblitzable examples is handy.
+  - High-level component patterns - We could have demo pages showing making menu bar, application shell, etc stitching together nimble-components. 
+  
+    This might require more discussion as to the benefit of what is being illustrated. Alternatively we should be offering those higher-level patterns as supported libraries.
+
+### Existing examples
+- The current [angular client example](https://github.com/ni/nimble/tree/main/angular-workspace/projects/example-client-app) and [blazor client example](https://github.com/ni/nimble/tree/main/packages/nimble-blazor/Examples)
+    - Will stay in current locations (angular-workspace and blazor workspace) as they are primarily developer oriented tools and not user facing examples
+    - Their READMEs should direct end users to the `<framework>` examples folders
+    - They will be renamed to `all-components-<framework>`
+    - Continue to not have a Vanilla all components example
+    - Note: This is a breaking URL change. For example, `https://nimble.ni.dev/storybook/example-client-app/` will change to `https://nimble.ni.dev/storybook/all-components-angular/`
+
+Alternatives:
+- Move the all-components examples to the examples/ folder
+    - Would hurt local development experience, i.e. need to run multiple angular builds / watch as a separate package not managed by angular-workspace
+    - Does not seem like a high value user facing example. The examples are very large and not illustrative of specific concepts.
+    - Users should rely on storybook for per control usage instead
+- Don't rename the current client examples
+  - Pros: Doesn't break existing URLs
+  - Cons: The current names are ambiguous about framework, i.e. `example-client-app` vs `blazor-client-app`. They also don't reflect the modern use-case of being a simple dev focused example showing all the components being used.
+
+## Alternative Implementations / Designs
+
+N/A
+
+## Open Issues
+
+### Publish examples to GitHub Pages
+
+Should we publish the examples to GitHub pages as well as make them accessible from stackblitz.
+
+Pros:
+- Easier to open on mobile devices
+- More reliable manual stress test environment (don't have to consider additional injected page content)
```

Comment:
>(don't have to consider additional injected page content)

Could you clarify what this means? Are you referring to stuff that Stackblitz adds to the page?

I have sometimes been annoyed when opening Stackblitz examples when they take more than a couple seconds to spin up their container. Having examples that avoid that would be nice. Obviously we have to consider the cost to implement it, but I like keeping it as part of the proposal.